Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -138,7 +137,7 @@ bool CVolumeDirectory::Read(u64 _Offset, u64 _Length, u8* _pBuffer, bool decrypt
 			if (_Length < fileBytes)
 				fileBytes = _Length;
 
-			if (!reader->Read(fileOffset, fileBytes, _pBuffer))
+			if (!file.Seek(fileOffset, SEEK_SET) || !file.ReadBytes(_pBuffer, fileBytes))
```

Comment:
Could we have less obfuscated control flow here? 
I find things that take advantage of short-circuited evaluation are *needlessly* harder to understand.

    if (!file.Seek(...))
        return false;
    if (!file.ReadBytes(...))
        return false;